src/main/java/frc/robot/RobotMap.java

```diff
@@ -2,8 +2,10 @@
 
 public class RobotMap {
 
-    private RobotMap() {}
+    public static final int UPPER_LIMIT_SWITCH = 1;
```

Comment:
```suggestion
    public static final int CORAL_ELEVATOR_UPPER_LIMIT_SWITCH = 1;
```
Name should be specific